### ISSUE TITLE
Need to update how types are exported

### ISSUE THREAD

**Issue by nikolasburk** [prisma/web#1033]
The info on this page needs to be updated:

https://www.prisma.io/docs/concepts/components/prisma-client/advanced-usage-of-generated-types

As of 2.12.0, the `ModelGetPayload` type is exported via the `Prisma` namespace:

Before

```ts
import { UserGetPayload } from '@prisma/client'
// Define a type that includes the relation to `Post`
type UserWithPosts = UserGetPayload<{
  include: { posts: true }
}>
// Define a type that only contains a subset of the scalar fields
type UserPersonalData = UserGetPayload<{
  select: { email: true; name: true }
}>
```

After

```ts
import { Prisma } from '@prisma/client'
// Define a type that includes the relation to `Post`
type UserWithPosts = Prisma.UserGetPayload<{
  include: { posts: true }
}>
// Define a type that only contains a subset of the scalar fields
type UserPersonalData = Prisma.UserGetPayload<{
  select: { email: true; name: true }
}>
```

**Comment by mhwelander** [comment]
@molebox Assigning this to you as it's part of the current issue you're working on

**Comment by molebox** [comment]
I think im going to refactor this to use the validator too.

See [this slack convo](https://prisma-company.slack.com/archives/C5Z9TH6N9/p1616410047084900?thread_ts=1612447130.099700&cid=C5Z9TH6N9)

**Comment by mhwelander** [comment]
Perfect 💯

**Comment by molebox** [comment]
Fixed in #1462
